Anchorage.swift

```diff
@@ -262,214 +264,221 @@ infix operator ~: PriorityPrecedence
     }
-
-    public func activeConstraintsGreaterThanOrEqualToEdges(_ anchor: EdgeAnchors?, constant c: CGFloat = 0.0, priority: UILayoutPriority = UILayoutPriorityRequired) -> EdgeConstraints {
+    
+    public func activeConstraints(greaterThanOrEqualToEdges anchor: EdgeAnchors?, constant c: CGFloat = 0.0, priority: UILayoutPriority = UILayoutPriorityRequired) -> EdgeConstraints {
         var constraintBuilder = ConstraintBuilder(horizontal: >=, vertical: >=)
-
+        
         if includedEdges.contains(.leading) && includedEdges.index(of: .trailing) == anchor?.includedEdges.index(of: .trailing) {
             constraintBuilder.trailing = (<=)
         }
-
+        
         if includedEdges.contains(.top) && includedEdges.index(of: .bottom) == anchor?.includedEdges.index(of: .bottom) {
             constraintBuilder.bottom = (<=)
         }
-
-        return constraintsForAnchors(anchor, constant: c, priority: priority, builder: constraintBuilder)
+        
+        return constraints(forAnchors: anchor, constant: c, priority: priority, builder: constraintBuilder)
     }
-
-    fileprivate subscript (edge: LayoutEdge) -> LayoutAnchorType {
+    
+    subscript (edge: LayoutEdge) -> LayoutAnchorType {
         switch edge {
         case .top:      return top
         case .leading:  return leading
         case .bottom:   return bottom
         case .trailing: return trailing
         }
     }
-
-    private func constraintsForAnchors(_ anchors: EdgeAnchors?, constant c: CGFloat, priority: UILayoutPriority, builder: ConstraintBuilder) -> EdgeConstraints {
+    
+    private func constraints(forAnchors anchors: EdgeAnchors?, constant c: CGFloat, priority: UILayoutPriority, builder: ConstraintBuilder) -> EdgeConstraints {
         guard let anchors = anchors else {
             preconditionFailure("Encountered nil edge anchors, indicating internal inconsistency of this API.")
         }
-
+        
         var edgeConstraints = EdgeConstraints()
-
+        
         zip(includedEdges, anchors.includedEdges).forEach { (edge, otherEdge) in
             edgeConstraints[edge] = {
                 switch (self[edge], anchors[otherEdge]) {
-
+                    
                 case let (x as NSLayoutXAxisAnchor, otherX as NSLayoutXAxisAnchor):
                     let expression = (otherX + edge.transform(constant: c)) ~ priority
-                    return builder.horizontalBuilderForEdge(edge)(x, expression)
-
+                    return builder.horizontalBuilder(forEdge: edge)(x, expression)
+                    
                 case let (y as NSLayoutYAxisAnchor, otherY as NSLayoutYAxisAnchor):
                     let expression = (otherY + edge.transform(constant: c)) ~ priority
-                    return builder.verticalBuilderForEdge(edge)(y, expression)
-
+                    return builder.verticalBuilder(forEdge: edge)(y, expression)
+                    
```

Comment:
There's lots of whitespace changes, and if I'm reading the diff right you added a bunch of leading whitespace. It'd be nice to get rid of this.